mewarpx/mewarpx/emission.py

```diff
@@ -689,7 +669,7 @@ def __init__(self, T, conductor=None, use_Schottky=True,
                 weights.  Defaults to True.
             emission_type (str): Distribution function type used to sample
                 velocities of the emitted particles. Must be defined in
-                metools.runtools.get_velocities. Defaults to 'thermionic'.
+                mewarpx.util.get_velocities. Defaults to 'thermionic'.
```

Comment:
This allows Sphinx to hyperlink the function.
```suggestion
                :func:`mewarpx.util.get_velocities`. Defaults to 'thermionic'.
```